core/modules/views/plugins/views_plugin_display_block.inc

```diff
@@ -70,20 +70,24 @@ class views_plugin_display_block extends views_plugin_display {
    */
   function execute_hook_block_list($delta = 0, $edit = array()) {
     $delta = $this->view->name . '-' . $this->display->id;
-    $info = $this->get_option('block_description');
+    $block_title = $this->get_option('block_description');
```

Comment:
It is unfortunate that this has been named block_description in config - it creates confusion.